tests/apollo/test_skvbc_long_running.py

```diff
@@ -50,7 +51,8 @@ class SkvbcLongRunningTest(unittest.TestCase):
                       selected_configs=lambda n, f, c: n == 7)
     async def test_stability(self, bft_network):
         bft_network.start_all_replicas()
-        with trio.move_on_after(seconds=EIGHT_HOURS_IN_SECONDS/2):
+        i=0
+        with trio.move_on_after(seconds=EIGHT_HOURS_IN_SECONDS):
             while True:
```

Comment:
Instead of having a separately maintained index `i`, let's use the following Python idiom for infinite indexed loop.
So we should replace:
```python
while True:
   ...
```
by:
```python
for i in itertools.count():
   ...
```